app/services/localization/languages/en.ts

```diff
@@ -364,6 +364,17 @@ export const en = {
       title:
         'use the slider to choose the emotion that best describes how you feel now.',
     },
+    instruction: {
+      first: {
+        title: 'A Quiet Space',
+      },
+      second: {
+        title: 'Bare Feet',
+      },
+      third: {
+        title: 'Sit down, or Stand Up',
+      },
+    },
```

Comment:
we don't have this file more
please make sure your translations synced with DatoCMS